src/components/auth/Login.jsx

```diff
@@ -1,7 +1,13 @@
+import FormInput from "../common/FormInput";
 const Login = () => {
   return (
     <>
       <h1>Login Form</h1>
+      {/* testing how the inputs looks like */}
+      <form>
+        <FormInput id="email" type="email" placeholder=" ">Email address</FormInput>
```

Comment:
Is passing an empty string for `placeholder` for both inputs a style choice to center the place holder text in the input? Removing the place holder attribute still shows the desired text except it is integrated into the border. Either is fine.

Reply:
Yes, without the placeholder attribute the label is into the border after first input focus.